### ISSUE TITLE
Playground: "An error has occurred" while dismissing transaction pop-up using ESC (not the "X link")

### ISSUE THREAD

**Issue by pfuntner** [hyperledger/composer#2091]
## Context

After deploying the digital property sample network in the composer playground and following the steps to create the participants/assets, and submit the transaction, I viewed the transaction.  After I tried to dismiss the transaction pop-up window, I got another pop-up that said _An error has occurred_.

## Expected Behavior

The transaction window should dismiss without any fuss.

## Actual Behavior

There is an extra pop-up alerting the user to some sort of anonymous error.  After dismissing that one, the playground acts normally.  I tried logging out of the network, back in, and viewing the transaction again and the same thing happened.

## Possible Fix

I don't know of a potential fix and don't even know the root cause of the _error_.

## Steps to Reproduce

1. I deployed the digital property sample network and followed the steps in `README.me` to create the participants and asset.
2. I submitted the `RegisterPropertyForSale` transaction
3. I clicked the transaction to view it:
![image](https://user-images.githubusercontent.com/23261559/30175463-9e8d8b72-93cc-11e7-9f38-78075604abba.png)
4. I dismissed the pop-up by pressing the ESC key and was presented with the new pop-up:
![image](https://user-images.githubusercontent.com/23261559/30175503-c00118e6-93cc-11e7-8ca5-80f3a14a2d7c.png)

## Existing issues

- [Stack Overflow issues](http://stackoverflow.com/tags/hyperledger-composer) - no similar issues
- [GitHub Issues](https://github.com/hyperledger/composer/issues) - no similar issues
- [Rocket Chat history](https://chat.hyperledger.org/channel/composer) - no similar issues

## Context

Everything seems to work fine so it's kind of an annoyance but I wonder if it's a sign of a real issue.

## Your Environment

* Version used: Playground v0.12.1
* Environment name and version (e.g. Chrome 39, node.js 5.4): Chrome Version 61.0.3163.79
* Operating System and version (desktop or mobile): Windows 7 Professional
* Link to your project: n/a

**Comment by pfuntner** [comment]
There is a _Close_ (X) link on the transaction pop-up and if I use that to close it instead of pressing ESC, there is no error pop-up.

**Comment by pfuntner** [comment]
I think this is happening for other sample networks too.  This may be the first time I've tried to use ESC to dismiss a transaction pop-up.

I think I've used ESC to dismiss other pop-ups such as one to create a new participant or asset.  I just tried this and it worked fine without producing an error pop-up.  There seem to be three ways to dismiss the other pop-ups without making a change (adding a participant/asset, etc.):

  1. Press _Cancel_ button
  2. Click the X link
  3. Press the ESC key
